axelor-sale/src/main/java/com/axelor/apps/sale/rest/dto/SaleOrderLinePostRequest.java

```diff
@@ -19,22 +19,17 @@
 package com.axelor.apps.sale.rest.dto;
 
 import com.axelor.apps.base.db.Product;
-import com.axelor.apps.sale.db.SaleOrder;
 import com.axelor.utils.api.ObjectFinder;
-import com.axelor.utils.api.RequestPostStructure;
+import com.axelor.utils.api.RequestStructure;
 import java.math.BigDecimal;
 import javax.validation.constraints.Min;
 import javax.validation.constraints.NotNull;
 
-public class SaleOrderLinePostRequest extends RequestPostStructure {
+public class SaleOrderLinePostRequest extends RequestStructure {
```

Comment:
we should not have the version in this query, since we are creating new lines, so they don't have a version.